src/domain/platform/metadata/useVersionControl.ts

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+
+const LOCALSTORATE_VERSIONCONTROL_KEY = 'releaseNotification';
+
+interface ReleaseNotificationData {
+  prevClientVersion: string;
+}
+
+const useVersionControl = () => {
+  const currentClientVersion = process.env.REACT_APP_VERSION ?? '';
+  const [isCurrentVersionViewed, setCurrentVersionViewed] = useState(false);
+
+  const saveCurrentVersionViewed = () => {
+    const data: ReleaseNotificationData = {
+      prevClientVersion: currentClientVersion,
+    };
+    localStorage.setItem(LOCALSTORATE_VERSIONCONTROL_KEY, JSON.stringify(data));
+    setCurrentVersionViewed(true);
+  };
+
+  useEffect(() => {
```

Comment:
good good. I also thought that it could happen the "open and immediately close", but all of this was behind a loading screen and didn't see it happening. But I agree, this is more clean. I also like the fact that they are synchronized between tabs.
Applied the changes, please take a look when you can.